packages/@react-spectrum/tree/docs/TreeView.mdx

```diff
@@ -419,49 +419,61 @@ The `<TreeViewItem>` component works with frameworks and client side routers lik
 
 ```tsx example
 <TreeView aria-label="Example tree with action groups" height="size-3000" maxWidth="size-6000" items={items}>
-  {(item: MyItem) => (
-    <TreeViewItem childItems={item.childItems} textValue={item.name}>
-      <TreeViewItemContent>
-        <Text>{item.name}</Text>
-        {item.icon}
-        <ActionGroup onAction={(key) => alert(`Item: ${item.id}, Action: ${key}`)}>
-          <Item key="edit" textValue="Edit">
-            <Edit />
-            <Text>Edit</Text>
-          </Item>
-          <Item key="delete" textValue="Delete">
-            <Delete />
-            <Text>Delete</Text>
-          </Item>
-        </ActionGroup>
-      </TreeViewItemContent>
-    </TreeViewItem>
-  )}
+  {function renderItem(item: MyItem) {
+    return (
+      <TreeViewItem childItems={item.childItems} textValue={item.name}>
```

Comment:
Is `childItems` needed here? It's also being passed into the Collection below. Actually looks like `childItems` is not even used in TreeView anymore. We should probably remove the prop from the API entirely?

Reply:
will try removing now

Reply:
one issue with removing it is that the types for custom TreeViewItem's will be a little harder, as people will need to include childItems

Reply:
not sure what you mean

Reply:
nevermind, i thought it was going to cause a type issue, but it appears we already set them